apps/mobile/app/utils/crashReporting.ts

```diff
@@ -23,46 +23,45 @@
  *  This is where you put your crash reporting service initialization code to call in `./app/app.tsx`
  */
 export const initCrashReporting = () => {
-  // Sentry.init({
-  //   dsn: "YOUR DSN HERE",
-  //   enableInExpoDevelopment: true,
-  //   debug: true, // If `true`, Sentry will try to print out useful debugging information if something goes wrong with sending the event. Set it to `false` in production
-  // })
-  // Bugsnag.start("YOUR API KEY")
+	Sentry.init({
+		dsn: process.env.EXPO_PUBLIC_SENTRY_DSN,
+		enableInExpoDevelopment: true,
+		debug: false, // If `true`, Sentry will try to print out useful debugging information if something goes wrong with sending the event. Set it to `false` in production
```

Comment:
@desperado1802 we should detect here if it's production or dev/stage envs etc. Please create GitHub issue to improve that, but don't work on it yet, it will require some more things from me to be done.

Reply:
Oh, actually @desperado1802 I notice we have already var __DEV__ (search code!) and you can use it to detect env (DEV or not dev). So debug should be enabled on DEV and disabled in any other env. Please fix that in that PR, no need to create GitHub issue!